docs/user-guide/settings/advanced.md

```diff
@@ -1,92 +1,161 @@
+The equivalent [config toggle](/getting-started/config-options/) is `PHOTOPRISM_DISABLE_BACKUPS`.
 
-#### Disable Places ####
-When selected, no geo-information is read from your files metadata. The Places section will not be visible.
+### Disable WebDAV
+This option prevents building WebDav connections.
+Requires restart for changes to be applied.
 
-#### Disbale ExifTool ####
-This option prevents the creation of json files with exif data in `storage/sidecar`. 
+The equivalent [config toggle](/getting-started/config-options/) is `PHOTOPRISM_DISABLE_WEBDAV`.
 
-#### Disable TensorFlow ####
-When selected, no labels are created for your files.
+### Disable Places
+When selected, no geo-information is read from your files metadata.
+The Places section will not be visible.
 
-## Image Quality ##
+The equivalent [config toggle](/getting-started/config-options/) is `PHOTOPRISM_DISABLE_PLACES`.
 
-You may change the settings for preview images in the *Advanced* settings tab (not available in public mode):
+### Disable ExifTool
+This option prevents the creation of `json` files with Exif data in `storage/sidecar`.
 
-![](img/advanced-settings.jpg)
+The equivalent [config toggle](/getting-started/config-options/) is `PHOTOPRISM_DISABLE_EXIFTOOL`.
 
-For best results, you should choose a high JPEG quality > 90, and the *lanczos* downscaling filter.
-This will require additional storage and slows down indexing, but only once when indexing for the first time.
-
-In comparison, the *cubic* filter may be 30% faster than *lanczos*.
-A [JPEG quality](http://fotoforensics.com/tutorial-estq.php) setting of 95 results in a preview files size of about 500kb.
-Reducing the JPEG quality to 80 gets storage size down to 100kb, and results in visible artifacts
-as in heavily compressed social media content.
-
-Image classification also works better with sharp images, 
-so it's likely you'll get less accurate labels with higher compression.
-
-## Preview Sizes ##
-
-Name      | Width  | Height  | Use                      |
-:---------|:------:|:-------:|:-------------------------|
-colors    | 3      | 3       | Color Detection          |
-tile_50   | 50     | 50      | List Preview             |
-tile_100  | 100    | 100     | Maps Preview             |
-tile_224  | 224    | 224     | Mosaic Preview           |
-left_224  | 224    | 224     | TensorFlow               |
-right_224 | 224    | 224     | TensorFlow               |
-tile_500  | 500    | 500     | Cards Preview            |
-fit_720   | 720    | 720     | Mobile, TV               |
-fit_1280  | 1280   | 1024    | Mobile, HD Ready TV      |
-fit_1920  | 1920   | 1200    | Mobile, Full HD TV       |
-fit_2048  | 2048   | 2048    | Tablets, Cinema 2K       |
-fit_2560  | 2560   | 1600    | Quad HD, Retina Display  |
-fit_3840  | 3840   | 2400    | Ultra HD                 |
-fit_4096  | 4096   | 4096    | Ultra HD, Retina 4K      |
-fit_7680  | 7680   | 4320    | 8K Ultra HD 2, Retina 6K |
```

Comment:
Why did you delete the table showing preview image sizes? Seems very important to understand what files will be created based on the settings...

Reply:
The list of thumbnail sizes is listed further down the page and this information is available via the 'developer docs' section which I've linked to, in case users wanted to dig deeper. Happy to restore this table if you like though. I do see some benefit for users knowing which files will be used where.